miranda/ecmwf/_era5.py

```diff
@@ -49,8 +48,9 @@ def request_era5(
 
     Parameters
     ----------
+    projects : str or List[str]
+      Allowed keys: {"era5-land", "era5-single-levels", "era5-single-levels-preliminary-back-extension", "era5-pressure-levels",  "era5-pressure-levels-preliminary-back-extension"}
```

Comment:
I understand the desire for this in our database, but ECMWF itself considers these as separate products. If we want to download the preliminary data, those strings are necessary. I still insist that we keep this information in some way (such as a `project` held in the metadata of files / a SQLite file) since the database structure doesn't allow for mixing these. 

I can sort the pressure level issue with some logic, but I'd like to be clear that the `preliminary-back-extension` is still preliminary and absolutely should not be mixed into other era5 products:
> An ERA5 back extension 1950-1978 (Preliminary version) has been produced. Although in many other respects the quality is relatively good, [this preliminary data does suffer from excessively intense tropical cyclones](https://confluence.ecmwf.int/display/CKB/ERA5+back+extension+1950-1978+%28Preliminary+version%29%3A+tropical+cyclones+are+too+intense). This dataset is available as a separate entry in the CDS catalogue (and in MARS) for a short period of time, after which it will be deprecated and replaced by a new updated version which will be accessible through the main ERA5 entry. The main entry currently contains data from 1979 onwards.

It also has multiple data quality issues specific only to the preliminary version: 
 - https://confluence.ecmwf.int/display/CKB/ERA5+back+extension+1950-1978+%28Preliminary+version%29%3A+tropical+cyclones+are+too+intense?src=contextnavpagetreemode
 - https://confluence.ecmwf.int/display/CKB/ERA5+back+extension+1950-1978+%28preliminary+version%29%3A+large+bias+in+surface+analysis+over+Australia+prior+to+1970?src=contextnavpagetreemode

Once the finalized version of ERA5 1950-1978 is made available (hopefully soon), we can disregard this. Does this seem reasonable?

Reply:
Yes, that's okay with me.